src/main/java/sorald/Processors.java

```diff
@@ -1,7 +1,29 @@
 package sorald;
 
 import java.util.Map;
-import sorald.processor.*;
+import sorald.processor.ArrayHashCodeAndToStringProcessor;
+import sorald.processor.BigDecimalDoubleConstructorProcessor;
+import sorald.processor.CastArithmeticOperandProcessor;
+import sorald.processor.CompareStringsBoxedTypesWithEqualsProcessor;
+import sorald.processor.CompareToReturnValueProcessor;
+import sorald.processor.DeadStoreProcessor;
+import sorald.processor.EqualsOnAtomicClassProcessor;
+import sorald.processor.GetClassLoaderProcessor;
+import sorald.processor.InterruptedExceptionProcessor;
+import sorald.processor.IteratorNextExceptionProcessor;
+import sorald.processor.MathOnFloatProcessor;
+import sorald.processor.PublicStaticFieldShouldBeFinalProcessor;
+import sorald.processor.SelfAssignementProcessor;
+import sorald.processor.SerialVersionUidProcessor;
+import sorald.processor.SerializableFieldInSerializableClassProcessor;
+import sorald.processor.SoraldAbstractProcessor;
+import sorald.processor.SynchronizationOnGetClassProcessor;
+import sorald.processor.SynchronizationOnStringOrBoxedProcessor;
+import sorald.processor.ThreadRunProcessor;
+import sorald.processor.ToStringReturningNullProcessor;
+import sorald.processor.UnclosedResourcesProcessor;
+import sorald.processor.UnusedThrowableProcessor;
+import sorald.processor.XxeProcessingProcessor;
```

Comment:
I think your IDE has been up to shenanigans here, this will cause the code generation checks to fail :)